cluster-autoscaler/proposals/buffers.md

```diff
@@ -0,0 +1,567 @@
+# Buffers API
+
+#### Author: Justyna Betkier (jbtk)
+
+# Summary
+
+If the user uses autoscaling the cluster size will be adjusted to the number of
+currently running pods. This means that often when a new pod is created its
+scheduling time will need to include the time of creating a new node and
+attaching it to the cluster. Many users care deeply about pod scheduling time
```

Comment:
There are a nontrivial amount of writes (events, etc) associated with evicting, deleting, creating, and scheduling pods. ETCD is pretty sensitive to writes -- I suspect there are pretty huge perf benefits to this if you were to measure it.